app/app.global.css

```diff
@@ -30,7 +32,7 @@ body {
 }
 
 p {
-  font-size: 24px;
+  font-size: 14px;
```

Comment:
If I understand correctly, the preferred way of changing base fonts if through settings when initializing material theme, not by overriding particular elements (in this case, paragraph):

```js
const theme = createMuiTheme({
  typography: {
    // In Japanese the characters are usually larger.
    fontSize: 12,
  },
});
```

https://material-ui.com/customization/themes/#typography-font-size